03/app.js

```diff
@@ -1,42 +1,51 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import Form from './Form';
 
 class Article extends React.Component {
     state = {
         comments: [],
-    }
-    
+        content: "",
+    };
+
+    handleChange = (e) => {
+        this.setState({ content: e.target.value });
+    };
```

Comment:
Ta cześć również mogłaby być w samym komponencie `Form` :) - nawet to rozwiązanie byłoby czytelniejsze.